### ISSUE TITLE
[Bug] Images inlined in html when running yarn build

### ISSUE THREAD

**Issue by alexjfno1** [react-static/react-static#1221]
### Environment

```
  System:
    OS: macOS High Sierra 10.13.6
    CPU: (4) x64 Intel(R) Core(TM) i5-2500S CPU @ 2.70GHz
    Memory: 624.45 MB / 12.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.15.3 - ~/.nvm/versions/node/v10.15.3/bin/node
    Yarn: 1.15.2 - ~/.nvm/versions/node/v10.15.3/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.15.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: 74.0.3729.169
    Firefox: 57.0.4
    Safari: 12.1.1
  npmPackages:
    react: ^16.8.2 => 16.8.6 
    react-dom: ^16.8.2 => 16.8.6 
    react-static: ^7.0.0 => 7.0.5 
    react-static-plugin-reach-router: ^7.0.0 => 7.0.0 
    react-static-plugin-sitemap: ^7.0.0 => 7.0.0 
    react-static-plugin-source-filesystem: ^7.0.0 => 7.0.0 
  npmGlobalPackages:
    react-static: 7.1.0

```

### Steps to Reproduce the problem

**Base your steps off of any freshly installed react-static template!**

1. `react-static create` -> basic template
2. Create a new file `imagespage.js` in `/pages` and add the following:
```javascript
import React from "react";
import Image from "../One.jpg";

export default function Images() {
  return <img src={Image} />;
}
```
3. Run `yarn build`
4. `ls -h dist/imagespage` to see that the `index.html` file is large in size which is due to the images being base64 encoded in the markup.

### Expected Behaviour

`yarn build` creates separate assets for the images which should be referenced in the `index.html` of that page instead of encoding the images directly.

**Comment by Pixelatex** [comment]
This does indeed seem to be standard behaviour with the latest RS version.

**Comment by alexjfno1** [comment]
The previous version of RS was fine with this so it's something that has changed in V7

**Comment by Pixelatex** [comment]
Alright, so I tinkered with it. I changed the file-loader to have a limit in the node build as well and it now outputs like this:

```<img src="/607fb0d20b1ab9f661009f6dc0ad731a.png"/>```

I'll make a PR for this

**Comment by SleeplessByte** [comment]
Fixed via #1222
